locust/argument_parser.py

```diff
@@ -420,6 +420,12 @@ def setup_parser_arguments(parser):
         help="User configuration as a JSON string or file. A list of arguments or an Array of JSON configuration may be provided",
         env_var="LOCUST_CONFIG_USERS",
     )
+    parser.add_argument(
+        "--run-users",
```

Comment:
I think `--user-classes` is a better name

Reply:
This creates an error since here https://github.com/locustio/locust/blob/137a269dccbb2fd0cb972aa25e5e4e40d74ee623/locust/test/test_load_locustfile.py#L169
you have short-form `u 100`, which gets parsed as `--u 100` instead of `-u 100 ` for some reason and so it gives an error because it cannot distinguish between `--users` and `--user-classes` if we use such a name

Reply:
Yea I figured what might cause an issue. I think --run-users is a bad name though. Can you think of something else?

Reply:
The best way would have been changing `--users` to `--num_users` and then adding `--user-classes`, but this is not backward-compatible.